### PR TITLE
Fix Chart.Show cross platform and target net472

### PR DESCRIPTION
**Pull request by zyzhu**
1. Updated Google.DataTable.Net.Wrapper to 4.0.0-rc1
2. `System.Diagnostics.Process.Start(path)` does not work on netcore app as it shall check which OS it runs and run differently on different OS. I changed target to net472 in order to run `System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform`
3. Updated other dependencies. Removed appveyor source for FSharp.Formatting.
4. Fixed dll paths for all scripts.

### CODE REVIEW

File: src/XPlot.Plotly/Main.fs [hunk 1]
```diff
@@ -40,7 +41,15 @@ module Html =
         let file = sprintf "%s.html" pageId
         let path = Path.Combine(tempPath, file)
         File.WriteAllText(path, html)
-        System.Diagnostics.Process.Start(path) |> ignore
+        if RuntimeInformation.IsOSPlatform(OSPlatform.Windows) then
```
Comment (jackfoxy) [inline src/XPlot.Plotly/Main.fs:44]: Why is this not a match?

**Review by jackfoxy — CHANGES_REQUESTED** [review]
Great job. Please respond to one and only comment. If you just want to change it to match...with I'll approve right away. Otherwise I'm always good to learn something new.

File: src/XPlot.Plotly/Main.fs (hunk 1 above)
Comment (zyzhu) [reply src/XPlot.Plotly/Main.fs:44]: `RuntimeInformation` only exposes API `IsOsPlatform` which is `OsPlatform -> bool`. Writing it with match ... with won't make it more simple.
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.isosplatform?view=netframework-4.8

**Comment by zyzhu** [comment]
I saw Linux travis build didn’t work. It had issue lately to get dotnet sdk. I saw discussion on Travis forum and changed it from trusty dist to xenial and solved the error in Deedle.

**Pull request merged**